CHANGELOG.md

```diff
@@ -1,3 +1,39 @@
+## 2.1.0 (October 20, 2015)
+### general
+ - Added ability to install and upgrade Logstash plugins without requiring internet connectivity (#2376). 
+ - Support alternate or private Ruby gems server to install and update plugins (#3576).
+ - Fixed a memory leak which could be triggered when events having a date were serialized to string (#4222).
+ - Added JDBC input to default package.
+ - Logstash can read multiple config files from a directory. If you combine --config_test CLI flag with 
```

Comment:
Maybe simpler this way?
`Adding --debug to --configtest now shows the configuration in blocks annotated by file. Very useful when using multiple config files in a directory.`